ldap/servers/slapd/daemon.c

```diff
@@ -826,7 +916,19 @@ accept_thread(void *vports)
     i_unix = ports->i_socket;
 #endif /* ENABLE_LDAPI */
 
+#ifdef ENABLE_EPOLL
```

Comment:
Similar to the above, I think shortterm duplicating the accept_threads() fn here is better than trying to ifdef so much which makes it harder to reader.

Reply:
Accepted as above.